com.yubico.YubiOath.json

```diff
@@ -0,0 +1,258 @@
+{
+    "app-id": "com.yubico.YubiOath",
+    "runtime": "org.kde.Platform",
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.12",
```

Comment:
Update runtime to 5.13

Reply:
That is not necessarily required. It'd be good to check.